### PR TITLE
[PoC] Add support for dynamic namespaces in SettingsBundle

### PR DESCRIPTION
**Pull request by stloyd**
Found this idea in my local repository, not sure it's working (probably not) but decided to show it ;)

### CODE REVIEW

File: src/Sylius/Bundle/SettingsBundle/Schema/SchemaRegistryInterface.php [hunk 1]
```diff
@@ -21,40 +21,43 @@
     /**
      * Get an array of all registered schemas.
      *
-     * @return array
+     * @return SchemaInterface[]
      */
     public function getSchemas();
```
Comment (aRn0D) [inline src/Sylius/Bundle/SettingsBundle/Schema/SchemaRegistryInterface.php]: @stloyd @pjedrzejewski Why don't we use the ServiceRegistry ?

File: src/Sylius/Bundle/WebBundle/Resources/views/Backend/Settings/oauth.html.twig [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+{% block topbar %}
+<ol class="breadcrumb">
+    <li>{{ 'sylius.breadcrumb.configuration'|trans }}</li>
+    <li>{{ 'sylius.breadcrumb.oauth_settings'|trans }}</li>
+</ol>
+{% endblock %}
+
+{% block content %}
+<div class="page-header">
+    <h1><i class="icon-cogs"></i> {{ 'sylius.settings.oauth_header'|trans({'%provider%': namespace|capitalize})|raw }}</h1>
```
Comment (aRn0D) [inline src/Sylius/Bundle/WebBundle/Resources/views/Backend/Settings/oauth.html.twig]: icon-cogs is not a bootstrap 3 icon.

**Comment by aRn0D** [comment]
@stloyd where do you call `isDynamic()` ? 
@pjedrzejewski I like it :+1:

**Comment by pamil** [comment]
Unfortunately, there's no SettingsBundle since v1.0.0-alpha.2

**Pull request closed without merging**